### PR TITLE
CLOUDP-83457: [mongocli] fix required flag for backup restore

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

_Jira ticket:_ CLOUDP-83457

## Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

## Further comments

This has probably been broken for a while, the flags were kind of a mess from back when we used to support Atlas as well

### CODE REVIEW

File: internal/cli/opsmanager/backup/restores_start.go [hunk 1]
```diff
@@ -182,33 +179,25 @@ func RestoresStartBuilder() *cobra.Command {
...
-			if e := opts.validateParams(); e != nil {
-				return e
-			}
-
 			return opts.Run()
 		},
 	}
 
 	cmd.Flags().StringVar(&opts.snapshotID, flag.SnapshotID, "", usage.SnapshotID)
-	// C/OM uses cluster ID
```
Comment (gssbzn) [inline internal/cli/opsmanager/backup/restores_start.go:200]: we don't support atlas anymore so this is not needed

File: internal/cli/opsmanager/backup/restores_start.go [hunk 2]
```diff
@@ -218,6 +207,6 @@ func RestoresStartBuilder() *cobra.Command {
 	cmd.Flags().StringVarP(&opts.Output, flag.Output, flag.OutputShort, "", usage.FormatOut)
 
 	_ = cmd.MarkFlagRequired(flag.ClusterID)
-	_ = cmd.MarkFlagRequired(flag.TargetClusterID)
```
Comment (gssbzn) [inline internal/cli/opsmanager/backup/restores_start.go:221]: this is only required for automatic restores

File: internal/cli/opsmanager/backup/restores_start.go [hunk 3]
```diff
@@ -159,10 +158,7 @@ func markRequiredAutomatedRestoreFlags(cmd *cobra.Command) error {
 		return err
 	}
 
-	if config.Service() == config.CloudService {
-		return cmd.MarkFlagRequired(flag.ClusterName)
-	}
```
Comment (gssbzn) [inline internal/cli/opsmanager/backup/restores_start.go:164]: not supported for a while now

**Review by andreaangiolillo — APPROVED** [review]
LGTM

**Pull request merged**